### DISCUSSION TITLE
Have Mason ask to install language server if none is detected for filetype?

### DISCUSSION THREAD

**Discussion by poperigby** [mason-org/mason.nvim#661] [Q&A]
It would be useful if I could get Mason to ask me if I want to install a language server for the current filetype, if there isn't one already installed. Is this something that can be done?

**Comment by williamboman** [comment]
Hello! Mason itself will probably never do something like this, but it sounds like it could be a nice (opt-in) feature for mason-lspconfig.

**Reply by poperigby** [reply]
That's true, it would probably be better for it to go in `mason-lspconfig`.
